frontend/src/scenes/feature-flags/JSONEditorInput.tsx

```diff
@@ -76,6 +82,11 @@ export function JSONEditorInput({
                     onChange?.(val)
                 }}
             />
+            {!focused && !value?.toString() && placeholder && (
```

Comment:
What was the problem with the placeholder again?

Reply:
There is none on the monaco editor 👻